x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/home/available_packages.tsx

```diff
@@ -291,6 +302,54 @@ export const AvailablePackages: React.FC = memo(() => {
       setSelectedCategory={setSelectedCategory}
       onSearchChange={setSearchTerm}
       showMissingIntegrationMessage
+      callout={eprPackageLoadingError || eprCategoryLoadingError ? <NoEprCallout /> : undefined}
     />
   );
 });
+
+function NoEprCallout() {
+  return (
+    <EuiCallOut
+      title={i18n.translate('xpack.fleet.epmList.eprUnavailableCalloutTitle', {
+        defaultMessage: 'Update your package registry setting to view more integrations\n',
+      })}
+      iconType="iInCircle"
+      color={'warning'}
+    >
+      <p>
+        <FormattedMessage
+          id="xpack.fleet.epmList.eprUnavailableCalloutTitleMessage"
+          defaultMessage="Kibana cannot reach the Elastic Package Registry, which provides Elastic Agent integrations. To view these integrations, set the URL for a {registryproxy} or an {onpremregistry}."
```

Comment:
The copy here doesn't feel right to me. I think we should make it match the terminology in the documentation. cc @mostlyjason @dborodyansky 

### Current
Kibana cannot reach the Elastic Package Registry, which provides Elastic Agent integrations. To view these integrations, set the URL for a registry proxy or an on-prem registry.

### Proposed
Kibana cannot reach the Elastic Package Registry, which provides Elastic Agent integrations. To view these integrations, configure a proxy server or host your own registry.

Reply:
cc @gchaps @dedemorton 

FYI - continued discussion from https://github.com/elastic/kibana/issues/115436#issuecomment-946858013

Reply:
++ to matching the terminology in the documentation

@thomasneirynck suggested in a conversation that maybe we don't need the current title.  Instead can use this for the title and description?  This seems a little easier to parse:   

Title: Kibana cannot reach the Elastic Package Registry, which provides Elastic Agent integrations
Description: To view these integrations, configure a proxy server or host your own registry.

Reply:
I agree. The change looks good!

Reply:
@dedemorton @gchaps @joshdover 

In another thread, we want to distinguish between the generic error message, and one due to faulty reponses from EPR.

This is what it looks now so far:

- EPR generic error message: This is the one we settled on in this thread
![image](https://user-images.githubusercontent.com/1833023/140805089-22eb219b-e639-4ada-9f64-edd65dfd1363.png)

- EPR 400/500 error: (this is proposal from my part)
![image](https://user-images.githubusercontent.com/1833023/140805275-a0226241-b24e-46bc-904b-2ec7268ca71c.png)

Reply:
I like your change. It's concise and clear.

Reply:
A couple of comments:

- Because the first line is a title, it doesn't need an ending period.
- In the first one:
   -  Shouldn't it be "encountered" instead of "encounters"?  Or could the title be a little simpler: "Kibana cannot connect to the Elastic Package Registry, which provides Elastic Agent integrations"
   - Remove "please" from the description and start with "Ensure" instead.

Reply:
Closing the loop on this. We ended up changing the generic wrt specific. Text-contents and messages the same. Please see https://github.com/elastic/kibana/pull/117598#pullrequestreview-801470613